### ISSUE TITLE
[Accessibility] Date mask: VoiceOver doesn't read out the digit right before the separator

### ISSUE THREAD

**Issue by luish** [nosir/cleave.js#402]
Problem with VoiceOver not reading out the digits followed by a forward slash.

## Example 1

![sep-19-2018 13-33-50](https://user-images.githubusercontent.com/133780/45768594-b01ea900-bc13-11e8-950a-584a3bb1c026.gif)

## Example 2

![sep-19-2018 13-56-46](https://user-images.githubusercontent.com/133780/45768665-ebb97300-bc13-11e8-8a09-325656a8ea74.gif)

Reproducible on both jsfiddle:
- Vanilla JS component: https://jsfiddle.net/nosir/kbaxx64s/
- React JS component: https://jsfiddle.net/nosir/gLLsrxxf/

**Comment by rogorman9** [comment]
@luish Any chance you were able to find a workaround for this? Still seems to be an issue in the latest version of Cleave.

**Comment by holvi-saravanan** [comment]
@luish @rogorman9 I also faced the same issue but in Vue. I have a clean solution. 

Use cleave programmatically. 
Use a regular input and bind the state var directly to input's value. Then on the input event call the below function.
`
const cleavingFunction = (value) => {
const myElem = document.getElementById('someid')
const cleave = new Cleave(myElem, {
  numeral: true,
 .....
})
cleave.setRawValue(223)
const cleavedVal = cleave.getFormattedValue()
this.inputVal = cleavedVal
}
`

This should work as expected.
